conda/cli/install.py

```diff
@@ -285,6 +285,10 @@ def install(args, parser, command='install'):
                     raise PackagesNotFoundError(e._formatted_chains, channels_urls)
 
         except (UnsatisfiableError, SystemExit, SpecsConfigurationConflictError) as e:
+            if not getattr(e, "allow_retry", True):
+                # TODO: This is a temporary workaround to allow downstream libraries 
+                # to inject this attribute set to False and skip the retry logic
```

Comment:
Can you elaborate here which error might be given that allow_retry and where it's implemented in conda-libmamba-solver?

Reply:
Done!